### PR TITLE
[spirv] more line info

### PR DESCRIPTION
**Pull request by jaebaek**

### CODE REVIEW

File: tools/clang/test/CodeGenSPIRV/spirv.debug.opline.precedence.hlsl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Run: %dxc -T cs_6_0 -E main -Zi
```
Comment (ehsannas) [inline tools/clang/test/CodeGenSPIRV/spirv.debug.opline.precedence.hlsl]: What is the purpose of this test? This test was not added to `CodeGenSpirvTest.cpp`

File: tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.vulkan1.1.hlsl [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Run: %dxc -T cs_6_0 -E main -Zi -fspv-target-env=vulkan1.1
```
Comment (ehsannas) [inline tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.vulkan1.1.hlsl]: This test was not added to `CodeGenSpirvTest.cpp`

File: tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.control.barrier.hlsl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// Run: %dxc -T cs_6_0 -E main -Zi
```
Comment (ehsannas) [inline tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.control.barrier.hlsl]: This test was not added in `CodeGenSpirvTest.cpp`

Comment (jaebaek) [reply tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.control.barrier.hlsl]: It was mistake. I just added it.

File: tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.vulkan1.1.hlsl (hunk 2 above)
Comment (jaebaek) [reply tools/clang/test/CodeGenSPIRV/spirv.debug.opline.intrinsic.vulkan1.1.hlsl]: added.

**Pull request merged**